### PR TITLE
feat(metrics): Introduce the ability to individually override the sample rate.

### PR DESCRIPTION
**Pull request by JTCunning**
I'll follow this up with a PR that adds individual sample rates to specific metrics.

### CODE REVIEW

**Review by mattrobenolt — APPROVED** [review]
For completeness, I’d add it to the `timer` decorator too.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]
Hold on, not good.

File: src/sentry/utils/metrics.py [hunk 1]
```diff
@@ -78,7 +75,8 @@ def worker():
 
         self._started = True
 
-    def incr(self, key, instance=None, tags=None, amount=1):
+    def incr(self, key, instance=None, tags=None, amount=1,
+             sample_rate=settings.SENTRY_METRICS_SAMPLE_RATE):
         if not self._started:
             self._start()
         self.q.put((key, instance, tags, amount))
```
Comment (mattrobenolt) [inline src/sentry/utils/metrics.py]: You need to pass `sample_rate` here too, otherwise, it’ll error when unpacking the tuple on L58.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]

File: src/sentry/utils/metrics.py (hunk 1 above)
Comment (JTCunning) [reply src/sentry/utils/metrics.py]: Completed via https://github.com/getsentry/sentry/pull/13431/commits/26672a734f4247dec76b6b942f04609e979f4707

**Review by mattrobenolt — APPROVED** [review]
I think you did it. Congratulations.

**Pull request merged**